paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -258,6 +242,29 @@ internal class PaymentSheetViewModel @Inject internal constructor(
             }
         }
 
+        viewModelScope.launch {
+            intentConfirmationHandler.state.collectLatest { state ->
+                when (state) {
+                    is IntentConfirmationHandler.State.Idle,
+                    // TODO(samer-stripe): Once we move all intent confirmation into the handler, we can rely on this.
```

Comment:
I'm starting to hate TODOs.

There's 2 types:
* The ones you'd change in a follow up regardless of if there's a TODO or not.
* The ones that are just left as litter forever.